05-Organizing-and-Executing-Simulations/04-Submitting-Cluster-Jobs.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>You can use <code>FlowProject.{pre,post}.true</code>  here as well even with aggregate operations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I wasn't aware that these exist. The will require some additional explanation for readers of the tutorial as the job document does not appear explicitly in <code>Project.pre.true("condition")</code>.</p>